topics/climate/tutorials/ocean_qcv_analysis/tutorial.md

```diff
@@ -42,308 +51,526 @@ This tutorial explains how to qualify and calibrate the nitrate sensor mounted o
 >
 {: .agenda}
 
-> <details-title>Short introduction on how Galaxy works</details-title>
+# Prerequisites
+
+## Galaxy 
+> <details-title>Galaxy account</details-title>
+> Create or login to your [Galaxy Europe account](https://earth-system.usegalaxy.eu/login/start).
+{: .details}
+
+> <details-title>Galaxy tips and tricks</details-title>
+> Once you are logged in, Galaxy is divided into 4 vertical panels:
+> - Far left: **Galaxy action list** (Upload, Tools, Interactive Tools…)

```

Comment:
Is this intended as the very first Galaxy tutorial people do? If not, you can omit these basics and just add one of the intro tutorials as a prerequisite